src/main/java/equipment/technology/TechnologyEquipmentFactory.java

```diff
@@ -0,0 +1,20 @@
+package equipment.technology;
+
+public class TechnologyEquipmentFactory {
+
+
+    public AbstractTechnologyEquipment getTechnologyEquipment(TechnologyEquipmentType technologyEquipmentType, String IP) {
+        switch (technologyEquipmentType) {
+            case VOIP:
+                return new Voip(IP);
+            case InternetComputer:
+                return new InternetComputer(IP);
+            case ArmyComputer:
+                return new ArmyComputer(IP);
+            default:
+                System.out.println("The equipment type: " + technologyEquipmentType.toString() + " does not exists");
+        }
+        throw new IllegalArgumentException("Invalid Equipment type");
```

Comment:
put this section in the `default` case- it makes the code's flow more readable at a quick glance